runtime/tcore.c

```diff
@@ -0,0 +1,114 @@
+#include "tcore.h"
+
+#include "lua.h"
+#include "lauxlib.h"
+
+#include "lobject.h"
+#include "lstate.h"
+#include "ltm.h"
+
+const char * titan_tag_name(int raw_tag)
```

Comment:
Since this isn't a generated file anymore I think we should be strict about code style.
`const char *titan_tag_name(int raw_tag)`